configure

```diff
@@ -18355,12 +18355,6 @@ $as_echo "$as_me: WARNING: unrecognized options: $ac_unrecognized_opts" >&2;}
 fi
 
 
-echo "creating Modules/Setup" >&6
-if test ! -f Modules/Setup
-then
-	cp $srcdir/Modules/Setup.dist Modules/Setup
```

Comment:
These few lines are the source of the 'evilness' with the current behavior.  Setup is created from Setup.dist but only on the first run of 'configure'.  If you update your source tree using the VCS, Setup.dist could get updated but Setup becomes stale.  Re-running 'configure' is not sufficient to fix it.  You have to do a "make distclean" or something dramatic.